components/FilAriane.tsx

```diff
@@ -5,39 +5,65 @@ import IconComponent, { IconName } from './ui/IconComponent'
 
 interface FilArianeProps {
   currentPath: string
+  currentRoute: string
 }
 
-export default function FilAriane({ currentPath }: FilArianeProps) {
-  const [ariane, setAriane] = useState<{ fragment: string; href: string }[]>([])
+export default function FilAriane({
+  currentPath,
+  currentRoute,
+}: FilArianeProps) {
+  const [ariane, setAriane] = useState<{ labelPath: string; href: string }[]>(
+    []
+  )
+
+  const routeToLabelPath: { [key: string]: string } = {
```

Comment:
```suggestion
  const routeToLabel: { [key: string]: string } = {
```
le label c'est l'intitulé/titre que l'on donne, le path c'est le chemin des pages, on distingue ces deux notions?